packages/fe/components/card.vue

```diff
@@ -94,6 +94,15 @@ export default {
         return this.variant === 'small' ? '0 0 69 69' : '0 0 46 46'
       }
       return '0 0 92 92'
+    },
+    cornerPath () {
```

Comment:
Yeah, as a quick note, the SVG spec does allow some attributes to be modified by CSS, but the ones that contain values that are dependent on the viewbox (or implied viewbox) are technically not _supposed_ to be modified by CSS. The `d` attribute for paths is probably the most complex viewbox interaction from an SVG attribute, so it makes sense that browsers wouldn't support its modification (except by JS)